src/VisualStudio/IntegrationTest/TestUtilities/InProcess2/EncapsulateField_InProc2.cs

```diff
@@ -21,6 +21,16 @@ public EncapsulateField_InProc2(TestServices testServices)
                 await TestServices.Editor.WaitForLightBulbSessionAsync();
             }
 
+            await ExecuteCommandAsync(WellKnownCommandNames.Refactor_EncapsulateField);
```

Comment:
📝 This is a second attempt to figure out why Encapsulate Field keyboard binding has no effect (first attempt was adding the call to `WaitForLightBulbSessionAsync`)